rbkit-lib/subscriber.cpp

```diff
@@ -55,6 +55,12 @@ void Subscriber::triggerGc() {
     commandSocket->sendCommand(triggerGC_Command);
 }
 
+void Subscriber::startCPUProfiling() {
+    RBKit::CmdStartProfile startCPUProfile;
```

Comment:
`start_cpu_profiling` and `stop_cpu_profiling` . We also need another command to set the type of clock to use (wall/cpu).